src/helpers/Move.js

```diff
@@ -0,0 +1,2 @@
+export const move = (move, position) => 
+    (move && position)
```

Comment:
Return type should be string not boolean, you can return just "" or FEN for initial position.
Other than that everything looks good.